src/main/java/roomescape/theme/controller/ThemeController.java

```diff
@@ -0,0 +1,44 @@
+package roomescape.theme.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import roomescape.theme.service.dto.request.ThemeRequest;
+import roomescape.theme.service.dto.response.ThemeResponse;
+import roomescape.theme.service.ThemeService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/themes")
+public class ThemeController {
+    private final ThemeService themeService;
+
+    public ThemeController(ThemeService themeService) {
+        this.themeService = themeService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<ThemeResponse>> getAllThemes() {
+        List<ThemeResponse> themes = themeService.getAllThemes();
+        return ResponseEntity.ok().body(themes);
```

Comment:
ResponseEntity가 제공하는 정적 팩토리 메서드들을 한번쯤 살펴보면 조금 더 간결한 코드로 동일한 응답을 제공할 수 있을 것 같아요! 😃 

```suggestion
        return ResponseEntity.ok(themes);
```

Reply:
좋네요! 반영했어요~

[refactor: ResponseEntity 정적 생성자 - 불필요한 body 메서드 호출 삭제](https://github.com/woowacourse/spring-roomescape-member/pull/208/commits/60005788a0357737831c0cc883e0fae8a14fef99)